ha/util/machine_id.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+
+
+from ha.execute import SimpleCommand
+from cortx.utils.log import Log
+
+class MachineId():
```

Comment:
Any reason to create the class for it?

Reply:
In case more functionality gets added here , having classes will be better .

Reply:
Ok. May be then, later, we will need to change the class name also. But yeah, Ok for now.